mender-auth/cli/keystore.cpp

```diff
@@ -70,11 +70,11 @@ error::Error MenderKeyStore::Load() {
 }
 
 error::Error MenderKeyStore::Save() {
-	if (!key_) {
+	if (!key_.Get()) {
```

Comment:
Not wrong, but maybe it's more idiomatic to provide

```
operator bool() {
    return Get() != nullptr;
}
```

in PrivateKey?

Reply:
Good suggestion